### ISSUE TITLE
Support for Firmware Version 16.0.0

### ISSUE THREAD

**Issue by ghost** [HookedBehemoth/ShareNX-Overlay#12]
The overlay doesn't work on the firmware version v16.0.0.

**Comment by HookedBehemoth** [comment]
[ovlShareNX.zip](https://github.com/HookedBehemoth/ShareNX-Overlay/files/10841362/ovlShareNX.zip)
Can you try this build?
Currently uploads won't work on corrupted prodinfo though. I don't know if I find the time to support that.

**Comment by ghost** [comment]
It doesn't crash anymore, but it says `Failed to init curl` when I try to use the Upload button.
![image](https://user-images.githubusercontent.com/66596892/221627570-34c3926a-0567-4cde-b500-e781580a671a.jpeg)

**Comment by HookedBehemoth** [comment]
Did you corrupt your prodinfo?

**Comment by ghost** [comment]
No, I don't think I did.
I use SysNAND with cfw, and my serial number is the same as the one under my Nintendo Switch. I am also able to go online.

**Comment by ghost** [comment]
Had the same problem, I fixed it by running
 `sudo dkp-pacman -S devkitA64 switch-tools switch-curl`

**Comment by Argyos** [comment]
> Had the same problem, I fixed it by running
> `sudo dkp-pacman -S devkitA64 switch-tools switch-curl`

Can share the build

**Comment by MetalikZX** [comment]
Hey there, can anyone share the working build?

**Comment by Cr4nkSt4r** [comment]
> Hey there, can anyone share the working build?

Sure.
[ovlShareNX.zip](https://github.com/HookedBehemoth/ShareNX-Overlay/files/11383672/ovlShareNX.zip)

**Comment by MetalikZX** [comment]
Thank you!

**Comment by Cr4nkSt4r** [comment]
> Thank you!

I've linked a new one, which should have the latest bug fix from libnx, so it should work in docked mode too.
Haven't got any time to test, but will later.

**[EDIT]**
Okay, tried it and it still has the curl init error, so I'll link my own compiled version again, but this time without the "fixed by Cr4nkSt4r" notice at the bottom.

**Comment by 91ajames** [comment]
> > Hey there, can anyone share the working build?
> 
> Sure. [ovlShareNX.zip](https://github.com/HookedBehemoth/ShareNX-Overlay/files/11383672/ovlShareNX.zip)

Thx man! The 1.14MB crash's the system. you're zip 796KB file works like a charm!

**Comment by Cr4nkSt4r** [comment]
> Is there any source code (the latest?)

Nope, I've deleted all my updated overlay stuff, because most devs updated the needed changes anyway, so there was no need on keeping it.
But all I did was updating the dependencies, maybe fixed some errors coming with that and that's it.
